src/main/java/racing/model/Names.java

```diff
@@ -0,0 +1,47 @@
+package racing.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Names {
+    public static final int MAX_NAME_LENGTH = 5;
+    List<String> names;
+
+    public Names(String[] names) throws IllegalAccessException {
+        validate(names);
+        this.names = Arrays.stream(names)
+                .collect(Collectors.toList());
+    }
+
+    public int size() {
+        return this.names.size();
+    }
+    public String name(int index){
```

Comment:
```suggestion
    }
    
    public String name(int index){
```

공백 또한 코딩 컨벤션이라고 생각합니다!